docker-compose.yml

```diff
@@ -154,8 +212,14 @@ services:
 
   mapstore:
     image: georchestra/mapstore:latest
+    healthcheck:
+      test: ["CMD-SHELL", "curl -s -f localhost:8080/mapstore/configs/config.json >/dev/null || exit 1"]
+      interval: 30s
+      timeout: 10s
+      retries: 10
     depends_on:
-      - database
+      database:
```

Comment:
missing dependency to LDAP too

Reply:
resolved in commit https://github.com/georchestra/docker/pull/173/commits/d6bbcabecc672e8c1ffccebbbeeda230982d7b56